### PR TITLE
fix: copying and add slack notifications 

### PR DESCRIPTION
**Pull request by tac0turtle**

### CODE REVIEW

File: .github/workflows/copy-md.yml [hunk 1]
```diff
@@ -8,20 +8,12 @@ on:
...
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v4
-        with:
-          ref: sync-docs-branch
-
-      - name: Checkout Sync-Branch
-        run: |
-          git fetch --all
-          git checkout sync-docs-branch
```
Comment (tac0turtle) [inline .github/workflows/copy-md.yml:24]: this branch is deleted so the job was failing, its redundant anyways

File: .github/workflows/copy-md.yml [hunk 2]
```diff
@@ -45,4 +37,46 @@ jobs:
...
+      - name: Notify Slack on success
+        if: ${{ steps.last_status.outputs.last_status == 'failure' }}
+        uses: rtCamp/action-slack-notify@v2.2.1
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          SLACK_CHANNEL: sdk-sims
+          SLACK_USERNAME: Sim Tests release/0.50.x
+          SLACK_ICON_EMOJI: ":white_check_mark:"
+          SLACK_COLOR: good
+          SLACK_MESSAGE: 0.50.x Sims are passing
```
Comment (julienrbrt) [inline .github/workflows/copy-md.yml:63]: This probably should be renamed

File: .github/workflows/copy-md.yml [hunk 3]
```diff
@@ -45,4 +37,46 @@ jobs:
...
+    steps:
+      - name: Notify Slack on failure
+        uses: rtCamp/action-slack-notify@v2.2.1
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          SLACK_CHANNEL: sdk-sims
+          SLACK_USERNAME: Sim Tests release/0.50.x
+          SLACK_ICON_EMOJI: ":skull:"
+          SLACK_COLOR: danger
+          SLACK_MESSAGE: 0.50.x Sims are failing
```
Comment (julienrbrt) [inline .github/workflows/copy-md.yml:81]: Ditto

**Review by julienrbrt — APPROVED** [review]
utACK

**Pull request merged**